api/grpcserver/mesh_service.go

```diff
@@ -147,11 +135,11 @@ func (s MeshService) getFilteredActivations(startLayer types.LayerID, addr types
 		return nil, status.Errorf(codes.Internal, "error retrieving activations data")
 	}
 	for _, atx := range atxs {
-		// Filter here, now that we have full data
-		if atx.Coinbase == addr {
+		if atx.PubLayerID.Uint64() >= startLayer.Uint64() {
```

Comment:
Why is the `.Uint64()` type conversion necessary here?

Reply:
good catch, it is not necessary